txpool/txpool.go

```diff
@@ -164,10 +165,18 @@ type TxPool struct {
 	// Event manager for txpool events
 	eventManager *eventManager
 
+	// struct which contains all whitelists
+	whitelists Whitelists
+
 	// indicates which txpool operator commands should be implemented
 	proto.UnimplementedTxnPoolOperatorServer
 }
 
+type Whitelists struct {
```

Comment:
I take it it's a single list not multiple. Name: `WhiteList`

Reply:
It's a struct which will contain multiple whitelists. Currently we have only one, but in future i expect to have more

Reply:
fixed 6974e01cbb5bd4d4c82039d9a5decc59aba85c87

Reply:
@dbrajovic ping